bunker_in_the_snow/map.conf

```
@@ -0,0 +1,18 @@
+team.1.pos = (103,-32,104)
+team.1.color = red
+team.2.pos = (-103,-32,-103)
+team.1 = red
+team.2 = blue
+team.2.color = blue
+
+initial_stuff = default:pick_stone,default:torch 10
+r = 115
+h = 140
+rotation = z
+base_node = ctf_map:stonebrick
+
+name = The Bunker In the Snow
+author = realyg, winniepee0, smugler5, Soulseeker, Darkf4antom and Tryhard_
+
+license = CC BY-SA 4.0
+others = Thanks to j45 for hosting the server to create the map.
```

Missing new line at the end.
```suggestion
others = Thanks to j45 for hosting the server to create the map.

```

---

That happens when you click `edit`. No need for that

---

It's better to add  it as some IDE's have the end of line set to `lf`, so it's better to add an endline to avoid issues.

---

What do you exactly mean?

---

It's always good to add an endline.

---

what is endline?

---

> what is endline?

_Offtopic_: A new line at the end of the file or an empty line.

---

Already committed!